src/plugins/notebook/components/PageCollection.vue

```diff
@@ -87,22 +87,26 @@ export default {
 
             const selectedPage = this.pages.find(p => p.isSelected);
             const defaultNotebook = getDefaultNotebook();
-            const defaultpage = defaultNotebook && defaultNotebook.page;
+            const defaultPageId = defaultNotebook && defaultNotebook.defaultPageId;
             const isPageSelected = selectedPage && selectedPage.id === id;
-            const isPageDefault = defaultpage && defaultpage.id === id;
+            const isPageDefault = defaultPageId === id;
```

Comment:
Since we're deleting the notebook page above on like 86, will `selectedPage && selectedPage.id === id` even match?
Maybe we need to move this check above the deletion.

Reply:
same